airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -79,7 +79,7 @@ class DAGDetailSchema(DAGSchema):
     timezone = TimezoneField()
     catchup = fields.Boolean()
     orientation = fields.String()
-    concurrency = fields.Integer()  # TODO: Remove in Airflow 3.0
+    concurrency = fields.Method("get_concurrency")  # TODO: Remove in Airflow 3.0
```

Comment:
The concurrency here is the same as `max_active_tasks` but deprecated. See https://github.com/apache/airflow/blob/fce46e603b742087a4b72aad42b3126b9c262aa1/airflow/models/dag.py#L388-L396

Reply:
Yeah that’s the point. `concurrency = fields.Integer()` triggers Marshmallow to access the deprecated `concurrency` property and emits a deprecation warning, and this change instead accesses `max_active_tasks` to avoid it. The additional `get_concurrency` method is added because Marshmallow is too smart to detect that `max_active_tasks` is already declared on the schema (right below this line) and throws an exception if we also directly access the same attribute for this field.

Reply:
Ok. 
You mean that in `get_concurrency` limit, if we use `obj.max_active_tasks` instead of `obj.max_active_runs`  it'll cause a problem?

Reply:
Oh… no, I meant if we use `concurrency = fields.Integer("max_active_tasks")` it’ll error. The `max_active_runs` below is a typo 🤦

Reply:
fixed